creusot/src/callbacks.rs

```diff
@@ -53,3 +94,16 @@ impl Callbacks for ToWhy {
         }
     }
 }
+
+pub fn get_body<'tcx>(tcx: TyCtxt<'tcx>, def_id: LocalDefId) -> Option<BodyAndPromoteds<'tcx>> {
```

Comment:
I think this is worth a small function level comment, explaining that it fetches from a thread local catch and will panic and will return `None` if called a second time.